lib/bin/main/WeatherSensorService/Public/WeatherSensorReaderType.kt

```diff
@@ -0,0 +1,25 @@
+//
+// SpinDance
+// Weather Sensor Service
+//
+// Copyright © 2022 SpinDance. All rights reserved.
+//
+
+import kotlinx.coroutines.flow.SharedFlow
+
+public interface WeatherSensorReaderType {
+     /// The time interval in seconds at which weather sensor readings are published
+     var readerInterval: UInt
```

Comment:
Since we have `fun set(readingInterval: UInt)`, it makes sense to make this immutable.

Reply:
Also, can you globally commonize the`readerInterval` vs `readingInterval` naming. I think I like the latter better.

Reply:
Wait a minute, I thought you got rid of the files in `bin`?